commune_app/all_models/votes.py

```diff
@@ -20,3 +20,6 @@ def create_new_vote(voting_user, voted_chore, vote_bool):
             if num_of_yes_votes > num_of_no_votes:
                 voted_chore.passed = True
                 voted_chore.save()
+
+    def vote(self, chore_id, decision):
+        Vote.create_new_vote(voting_user=self.id, voted_chore=chore_id, vote_bool=decision)
```

Comment:
1. maybe you can change the Vote to self
2. I think you need to get the voting_user because the self.id is the id of the vote object
```suggestion
        self.create_new_vote(voting_user=self.id, voted_chore=chore_id, vote_bool=decision)
```